examples/llm/src/models/layers/attention.py

```diff
@@ -366,20 +368,24 @@ def forward(self,
         return self.out_proj(context), attn_weights, past_key_value
 
 
-def attn_bias_shape(attn_impl, n_heads, seq_len, alibi, causal):
+def attn_bias_shape(attn_impl, n_heads, seq_len, alibi, prefix_lm, causal):
     if attn_impl == 'flash':
         return None
     elif attn_impl == 'triton':
         if alibi:
             if not causal:
                 return (1, n_heads, seq_len, seq_len)
             return (1, n_heads, 1, seq_len)
+        elif prefix_lm:
+            return (1, n_heads, seq_len, seq_len)
         return None
     elif attn_impl == 'torch':
         if alibi:
             if not causal:
                 return (1, n_heads, seq_len, seq_len)
             return (1, n_heads, 1, seq_len)
+        elif prefix_lm:
+            return (1, n_heads, seq_len, seq_len)
         return None
```

Comment:
Squinting hard here... are both these blocks the same? Can we just do `elif attn_impl in ['triton', 'torch']`

(this is also from a previous PR so not your fault)